### PR TITLE
Feature/rename documentation

### PR DESCRIPTION
**Pull request by shuchitak**
https://github.com/xmos/sw_avona/issues/309

Replaced Avona references in the documentation with Voice Framework.

### CODE REVIEW

File: doc/user_guide/audio_processing/index.rst [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
 
-A flexible audio signal routing infrastructure and a range of digital inputs and outputs enables the Avona Reference Design to be integrated into a wide range of system configurations, that can be configured at start up and during operation through a set of control registers.  In addition, all source code is provided to allow for full customization or the addition of other audio processing algorithms.
+A flexible audio signal routing infrastructure and a range of digital inputs and outputs enables the Voice Framwork to be integrated into a wide range of system configurations, that can be configured at start up and during operation through a set of control registers.  In addition, all source code is provided to allow for full customization or the addition of other audio processing algorithms.
```
Comment (mbanth) [inline doc/user_guide/audio_processing/index.rst:7]: Mis-spelling of 'Framework'

File: index.rst [hunk 2]
```diff
@@ -1,6 +1,8 @@
-#################################
-XMOS Avona Voice Reference Design
-#################################
+################
+Voice Framework
```
Comment (mbanth) [inline index.rst:2]: The README.rst file uses the title 'XMOS Voice Framework DSP Library'.  I do not see a need to use exactly the same name everywhere, but it does make me wonder if this line should be 'XMOS Voice Framework'.

File: settings.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Avona Voice Reference Design",
-    "project": "Avona",
+    "title": "XCORE Voice Framework",
```
Comment (mbanth) [inline settings.json:2]: `XCORE` or `XMOS`?  I don't much care one way or the other, but right now both are in use and I do not think that the inconsistency is a good idea.

**Review by mbanth — CHANGES_REQUESTED** [review]

File: index.rst (hunk 2 above)
Comment (shuchitak) [reply index.rst:2]: I referred to some other framework libraries and they weren't using XMOS either. It was IO Framework, Core Framework etc. so I think just Voice Framework follows the convention. Also, it is the xmos/fwk_voice repo so don't think specifying XMOS as part of the name is necessary. Let me know if you want me to change it. Thanks

File: settings.json (hunk 3 above)
Comment (shuchitak) [reply settings.json:2]: XCORE is used in the title for IO and Core frameworks so decided to follow the same convention.

**Comment by tomblackie-xmos** [comment]
The top level naming -    Voice Framework vs XCORE-Voice Framework etc - needs to be reviewed by @jacobh-xmos

**Review by mbanth — APPROVED** [review]
I have requested that @jacobh-xmos review as well to verify that the documentation meets the naming requirements from PM.

**Comment by mbanth** [comment]
@jacobh-xmos and I have determined that he does not currently have sufficient permissions to use the GitHub mechanism to approve this pull request.  He has approved it verbally.  @shuchitak, please go ahead and merge this pull request and take any normal subsequent actions related to merging it.

**Pull request merged**